main.cpp

```diff
@@ -198,6 +223,8 @@ struct School
         void teachClass(std::string day, int classDuration = 60);
         bool askForPromotion(double payIncrease);
         void dismissClass(int minRemaining = 0, bool finishedLecture = true);
+        // new member function
+        Teacher returnTeacher(int threshold, int startingValue);
```

Comment:
a Teacher instance will return another Teacher?
that doesn't make much sense.
This is a member function, so the only way to call it is by using a Teacher instance...
- [ ] revise